### ISSUE TITLE
Eliminate oauth2client dependency

### ISSUE THREAD

**Issue by nworden** [google/personfinder#500]
[It's deprecated.](https://pypi.org/project/oauth2client/) They recommend replacing it with [google-auth](https://google-auth.readthedocs.io/en/latest/) and [oauthlib](https://oauthlib.readthedocs.io/en/v3.0.1/).

**Comment by gimite** [comment]
I knew that it's deprecated, but I needed to use it since google-auth doesn't support Python 2. So this issue is rather blocked by Python 3 migration.

**Comment by nworden** [comment]
Yeah, there's a number of places where I think we're going to end up needing two codepaths (one for Python 2 and one for Python 3) while we migrate.

**Comment by nworden** [comment]
It supports up to 3.5; AE's Python 3 runtime is 3.7. If we can verify that it still works in 3.7, then it seems ok to keep using it, but it might be safer to switch away from it.
I'm pretty sure we can import it conditionally to avoid problems in Python 2 if we have to.

**Comment by gimite** [comment]
Maintaining two codepaths can be pain anyway, so I would prefer to keep using oauth2client for now if it works (especially if that's the only option for our production (Python 2)), but it's up to you.
